tests/subsys/net/lib/nrf_cloud/client_id/boards/qemu_cortex_m3.conf

```diff
@@ -0,0 +1,14 @@
+#
```

Comment:
if the test must run on a DK, why do we have this board file?

Reply:
I'll remove it. It's left over from when I was trying to get it to work.